src/VisualStudio/Core/Def/ProjectSystem/VisualStudioProjectFactory.cs

```diff
@@ -74,7 +74,9 @@ public Task<VisualStudioProject> CreateAndAddToWorkspaceAsync(string projectSyst
                 ? filePath
                 : null;
 
-            await _visualStudioWorkspaceImpl.EnsureDocumentOptionProvidersInitializedAsync(cancellationToken).ConfigureAwait(true);
+            // After here, everything can be off the UI thread.
```

Comment:
Nittiest of nits: Make this "After EnsureDocumentOptionProviders" just in case someone sees this, and thinks it means "After this comment", and moves the thread switch up a line.

Reply:
Oh, right, good point.